Site/bc_weather_station_dashboard/website/templates/weather.html

```diff
@@ -152,9 +157,29 @@ <h2 class="card-text" id="snow-quality">N/A</h2>
                     <!-- Wind Speed Card -->
                     <div class="col-md-4">
                         <div class="card foreground">
+                            <div class="row justify-content-center align-items-center mb-0">
+                                <div class="col-auto">
+                                    <img src="{% static 'images/wind-icon.svg' %}" alt="Wind Speed Icon"
```

Comment:
Want to add this to the left of the title like the temp and humidity widget?

Reply:
Couldn't get that to work, maybe could use some help with it tmr in class